### PR TITLE
sort taskRuns in get_logs by startTime, as tasks aren't ordered

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-9953

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: tests/test_tekton.py [hunk 1]
```diff
@@ -758,10 +858,34 @@ def test_get_logs_stream(self, pipeline_run):
...
+                         'Bye World'),
+                        (PIPELINE_RUN_JSON['status']['taskRuns']
+                         [TASK_RUN_NAME2]['pipelineTaskName'],
+                         '2Hello World'),
+                        (PIPELINE_RUN_JSON['status']['taskRuns']
+                         [TASK_RUN_NAME2]['pipelineTaskName'],
+                         '2'),
+                        (PIPELINE_RUN_JSON['status']['taskRuns']
+                         [TASK_RUN_NAME2]['pipelineTaskName'],
+                         '2Bye World')]
```
[the remark this answers is not in the document]
Comment (rcerven) [reply tests/test_tekton.py:891]: have you seen those different startTimes for the tasks??

[the remark this answers is not in the document]
Comment (chmeliik) [reply tests/test_tekton.py:914]: Oh OK, I mixed this up with the other test. My bad :smile:

File: osbs/tekton.py [hunk 2]
```diff
@@ -52,6 +54,12 @@ def check_response(response, log_level=logging.ERROR):
         raise OsbsResponseException(message=content, status_code=response.status_code)
 
 
+def get_sorted_task_runs(task_runs: Dict[str, Any]) -> Dict[str, Any]:
```
Comment (chmeliik) [inline osbs/tekton.py:57]: I think this returns a list of tuples
```python
In [1]: d = {'a': 1, 'b': 2}
In [2]: print(sorted(d.items()))
[('a', 1), ('b', 2)]
```
```suggestion
def get_sorted_task_runs(task_runs: Dict[str, Any]) -> List[Tuple[str, Dict[str, Any]]]:
```

**Review by chmeliik — COMMENTED** [review]
LGTM, minor typing issue

File: osbs/tekton.py (hunk 2 above)
Comment (rcerven) [reply osbs/tekton.py]: fixed

**Comment by rcerven** [comment]
rn not updated, fix for unreleased feature

**Pull request merged**